New Grafana dashboards docs + InfluxDB update (#1161)
grafana/k6-docs: src/data/markdown/translated-guides/en/04 Results output/300 Grafana dashboards.md

---
title: Grafana dashboards
excerpt: With multiple k6 output formats, you also have multiple ways to visualize test results in a Grafana dashboard.
---

You have multiple ways to query k6 results in Grafana. 
Having test results in a dashboard brings various benefits:
- Visualize your results to analyze performance during the test run or over multiple test runs. 
- Correlate test results with application and system metrics in the same dashboard to get a holistic overview of your system's performance and quickly find the root causes of performance issues. 

![A grafana dashboard correlating k6 results with observability data](./images/correlated-grafana-dashboard-grafana-cloud-k6.png)

## Options

With [Grafana](https://grafana.com/grafana/), you can create a custom dashboard to query and **visualize data from multiple sources and any type of backend**. With k6, you can stream your test results to any backend using a [custom output extension](/extensions/get-started/create/output-extensions/) or an existing [real-time output](/results-output/real-time/#service).

> The flexibility and interoperability of Grafana and k6 allow you to dashboard test and observability data together, independently of where the data is stored.

Author: ```suggestion
The flexibility and interoperability of Grafana and k6 let you join test and observability data in one dashboard, independently of where the data is stored.
```
